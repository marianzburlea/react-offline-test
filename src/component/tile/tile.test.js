import React from 'react'
import { create } from 'react-test-renderer'
import Tile from '.'

describe('@Tile', () => {
  it('should match the snapshot', () => {
    const tile = create(<Tile fuel="solar" perc={100} />)
    expect(tile.toJSON()).toMatchSnapshot()
  })
})
