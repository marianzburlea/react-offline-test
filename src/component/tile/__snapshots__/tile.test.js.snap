// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@Tile should match the snapshot 1`] = `
<div
  className="sc-htpNat kFsZpb"
>
  <div
    className="sc-bwzfXH bkXHvI"
  >
    solar
  </div>
  <div
    className="sc-bdVaJa fuxuhi"
  >
    100
  </div>
  <div />
</div>
`;
